kroxylicious-docs/docs/_modules/sasl-inspection/ref-glossary.adoc

```diff
@@ -8,4 +8,6 @@ Glossary of terms used in the SASL Inspection guide.
 JWE:: JSON Web Encryption is an IETF https://datatracker.ietf.org/doc/html/rfc7516[standard] for exchanging encrypted data using JSON.
 JWT:: JSON Web Token is an IETF https://datatracker.ietf.org/doc/html/rfc8259[standard] for securely transmitting information between parties as a JSON object.
 JWS:: JSON Web Signature is an IETF-proposed https://datatracker.ietf.org/doc/html/rfc7515[standard] for signing arbitrary data.
-SASL:: Simple Authentication and Security Layer, a framework for handling authentication.
\ No newline at end of file
+Principal:: A component of a subject.
```

Comment:
It might be good to explicitly state a Subject can have multiple Principals somewhere

Reply:
Yeah.  It is a bit of an aspirational statement at the minute.  It is not actually true. `User` is annotated `@Unique` and User is our only Principal type.  So at the moment, Subject can have exactly one.

I'm not proposing to change the glossary.